### DISCUSSION TITLE
[Feature Request] Accordion controlled

### DISCUSSION THREAD

**Discussion by jsdev-ninja** [heroui-inc/heroui#4713] [Feature requests]
### Is your feature request related to a problem? Please describe.

implement accordion controlled mode with react key is bad

should be changed to regular state with or similar solution

### Describe the solution you'd like

pls replace accordion state without using react key

### Describe alternatives you've considered

pls replace accordion state without using react key

### Screenshots or Videos

pls replace accordion state without using react key

**Comment by linear** [comment]
<p><a href="https://linear.app/nextui-inc/issue/ENG-1730/[feature-request]-accordion-controlled">ENG-1730 [Feature Request] Accordion controlled</a></p>
